src/specs/specs-css.json

```diff
@@ -1,52 +1,59 @@
 [
   "https://compat.spec.whatwg.org/",
-  "https://drafts.csswg.org/css-animations-2/",
-  "https://drafts.csswg.org/css-backgrounds-4/",
+  "https://drafts.css-houdini.org/box-tree-api/",
+  "https://drafts.css-houdini.org/css-typed-om-2/ delta",
+  "https://drafts.css-houdini.org/font-metrics-api/",
+  "https://drafts.csswg.org/css-animations-2/ delta",
+  "https://drafts.csswg.org/css-backgrounds-4/ delta",
   "https://drafts.csswg.org/css-box-3/",
-  "https://drafts.csswg.org/css-color-5/",
-  "https://drafts.csswg.org/css-conditional-4/",
+  "https://drafts.csswg.org/css-conditional-4/ delta",
   "https://drafts.csswg.org/css-env-1/",
   "https://drafts.csswg.org/css-exclusions-1/",
-  "https://drafts.csswg.org/css-gcpm-4/",
-  "https://drafts.csswg.org/css-multicol-2/",
+  "https://drafts.csswg.org/css-gcpm-4/ delta",
```

Comment:
Is the "is delta" bit something we'll need to maintain in https://github.com/w3c/browser-specs/? If so can you comment on https://github.com/w3c/browser-specs/issues/1?

Reply:
Yes. As far as I can tell, this is what we need to track the notion of "current level" that was raised in w3c/browser-specs#5. That is, all specs in Shepherd for which the "current level" is not the latest level are delta specs in practice. [`css-break-4`](https://drafts.csswg.org/css-break-4/) seems to be one exception to the rule, and I don't quite understand which it's not considered to be the "current level". Anyway, I'll comment over there.